modules/ROOT/pages/distributed-session-caching.adoc

```diff
@@ -88,10 +89,13 @@ public class ServiceB extends HttpServlet {
 ----
 
 Each JCache implementation configures the cluster instances differently.
-See the documentation for your chosen implementation.
+For more information, see the documentation for your chosen implementation.
 
-You don’t need to specify a Hazelcast configuration file if the default Hazelcast configuration works for you.
-However, you can create configuration file similar to the following example and add a reference to it from your `server.xml` file to customize how Hazelcast defines the cluster:
+== Customize your JCache implementation
+
+Most JCache implementations provide default settings.
```

Comment:
I think it's likely that all implementations provide default settings

I think you should say something like:
`Most JCache implementations provide sensible default settings.`

doesn't have to be "sensible", though there is precedent for that term. vis-a-vis "[convention over configuration](https://en.wikipedia.org/wiki/Convention_over_configuration)"

The idea to convey is that in many cases you don't need to customize the settings, so users know this is not necessarily required

Reply:
updated.